.env

```diff
@@ -0,0 +1,50 @@
+# Set all your environment variables here
+
+
+#########################################
+## GENERAL
+#########################################
+
+HOSTNAME=localhost
+# Adapt to your production domain and use a reverse proxy
+#HOSTNAME=taiga.company.com
+
+PORT=80
```

Comment:
@AminovE99 For developments, it's better to use a different port like `8080` as `80` is often already used by Apache or NGinx.